<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBShower</class>
 <widget class="QWidget" name="DBShower">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1532</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBShower</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(234, 234, 234);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,7">
   <item>
    <widget class="QWidget" name="choosewidget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1,1,1,1,1,5">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnMachine">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(246, 255, 231);border: 1px solid;border-radius: 20px;
</string>
        </property>
        <property name="text">
         <string>折弯机</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/img/machine.ico</normaloff>:/img/machine.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRobot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(246, 255, 231);border: 1px solid;border-radius: 20px;
</string>
        </property>
        <property name="text">
         <string>机器人</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/img/robot2.ico</normaloff>:/img/robot2.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnCenter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(246, 255, 231);border: 1px solid;border-radius: 20px;
</string>
        </property>
        <property name="text">
         <string>对中台</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/img/center2.ico</normaloff>:/img/center2.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnFlip">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(246, 255, 231);border: 1px solid;border-radius: 20px;
</string>
        </property>
        <property name="text">
         <string>翻面架</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/img/flip2.ico</normaloff>:/img/flip2.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnUp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(246, 255, 231);border: 1px solid;border-radius: 20px;
</string>
        </property>
        <property name="text">
         <string>上料台</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/img/Up.ico</normaloff>:/img/Up.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnDown">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(246, 255, 231);border: 1px solid;border-radius: 20px;
</string>
        </property>
        <property name="text">
         <string>下料台</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/img/Down.ico</normaloff>:/img/Down.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTableWidget" name="Machinetable">
               <property name="font">
                <font>
                 <family>楷体</family>
                </font>
               </property>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>180</number>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>60</number>
               </attribute>
               <row>
                <property name="text">
                 <string>1</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>2</string>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>名称</string>
                </property>
                <property name="whatsThis">
                 <string>name</string>
                </property>
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="icon">
                 <iconset resource="resource.qrc">
                  <normaloff>:/img/Setting.ico</normaloff>:/img/Setting.ico</iconset>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>模型文件地址</string>
                </property>
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="icon">
                 <iconset resource="resource.qrc">
                  <normaloff>:/img/datashow.ico</normaloff>:/img/datashow.ico</iconset>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>实例图片地址</string>
                </property>
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="icon">
                 <iconset resource="resource.qrc">
                  <normaloff>:/img/JPG_icon.png</normaloff>:/img/JPG_icon.png</iconset>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>创建时间</string>
                </property>
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
               </column>
               <item row="0" column="0">
                <property name="text">
                 <string>adaw</string>
                </property>
               </item>
               <item row="0" column="1">
                <property name="text">
                 <string>sadaw</string>
                </property>
               </item>
               <item row="0" column="2">
                <property name="text">
                 <string>awda</string>
                </property>
               </item>
               <item row="0" column="3">
                <property name="text">
                 <string>awda</string>
                </property>
               </item>
               <item row="1" column="0">
                <property name="text">
                 <string>sada</string>
                </property>
               </item>
               <item row="1" column="1">
                <property name="text">
                 <string>fdsfd</string>
                </property>
               </item>
               <item row="1" column="2">
                <property name="text">
                 <string>dsfsd</string>
                </property>
               </item>
               <item row="1" column="3">
                <property name="text">
                 <string>sfds</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_5" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>16</pointsize>
                    <kerning>true</kerning>
                   </font>
                  </property>
                  <property name="contextMenuPolicy">
                   <enum>Qt::DefaultContextMenu</enum>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(229, 229, 229);</string>
                  </property>
                  <property name="text">
                   <string>新增数据</string>
                  </property>
                  <property name="icon">
                   <iconset resource="resource.qrc">
                    <normaloff>:/img/add.ico</normaloff>:/img/add.ico</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>false</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(229, 229, 229);</string>
                  </property>
                  <property name="text">
                   <string>删除数据</string>
                  </property>
                  <property name="icon">
                   <iconset resource="resource.qrc">
                    <normaloff>:/img/delete.ico</normaloff>:/img/delete.ico</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="popupMode">
                   <enum>QToolButton::DelayedPopup</enum>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_4" native="true"/>
             </item>
             <item>
              <widget class="QWidget" name="widget_6" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>show;lab</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
